app/views/games/index.html.erb

```diff
@@ -1,182 +1,51 @@
+          <br class="clr" />
+          <h4>Available game<a class="btn btn-success pull-right" href="#" role="button">Join game!</a></h4>
       </div>
     </div>
-  </section>
-  <hr />
-  <section id="features">
-    <div class="row">
-      <div class="col-sm-offset-2 col-sm-8 text-center">
-        <h2>CHESS APP - FINAL PROJECT</h2>
-        <h4>Description of the project and fact that it is our final project for FHP</h4>
+  </section>          
+  <section id="profile">
+    <div class-"row">
+      <br />
+      <div class="col-sm-offset-2 col-sm-1">
+        <a class="btn btn-info dash-button" href="<%= user_path(current_user) %>" role="button">User Dashboard</a>
       </div>
-    </div>
-    <div class="row">
-      <div class="col-sm-offset-2 col-sm-2 col-xs-offset-2 col-xs-8 text-center"/>
-        <%= image_tag 'chess.jpg', class: 'img-responsive feature-pic' %>
-        <h3>Feature 1</h3>
-        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ligula leo, auctor non tincidunt sit amet, elementum vel dolor. Praesent convallis turpis et justo convallis, a rhoncus quam volutpat.</p>
+      <div class="col-sm-offset-2 col-sm-2">
+        <a class="center-block btn btn-success btn-lg dash-button" href="#" role="button">Create a Game</a>
       </div>
-      <div class="col-sm-offset-1 col-sm-2 col-xs-offset-2 col-xs-8 text-center"/>
-        <%= image_tag 'chess.jpg', class: 'img-responsive feature-pic' %>
-        <h3>Feature 2</h3>
-        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ligula leo, auctor non tincidunt sit amet, elementum vel dolor. Praesent convallis turpis et justo convallis, a rhoncus quam volutpat.</p>
-      </div>
-      <div class="col-sm-offset-1 col-sm-2 col-xs-offset-2 col-xs-8 text-center"/>
-        <%= image_tag 'chess.jpg', class: 'img-responsive feature-pic' %>
-        <h3>Feature 3</h3>
-        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ligula leo, auctor non tincidunt sit amet, elementum vel dolor. Praesent convallis turpis et justo convallis, a rhoncus quam volutpat.</p>
-      </div>
-    </div>
-  </section>
-  <hr />
-  <section id="team">
-    <div class="row">
-      <div class="text-center">
-        <h2>THE TEAM</h2>
+      <div class="col-sm-offset-2 col-sm-1">
+        <a class="btn btn-info" href="#" role="button">Account Settings</a>
       </div>
     </div>
+    <br class="clr" />
     <div class="row">
-      <div class="col-sm-2 col-sm-offset-1 col-xs-offset-2 col-xs-8">
-        <%= image_tag 'chad.png', class: 'img-responsive team-profile' %>
-        <h5 class="text-center">Chad</h5>
-          <div class="text-center">
-          <%= image_tag 'linked_in.png' %> &emsp;
-          <%= link_to 'https://github.com/chadbaum' do %>
-            <%= image_tag 'github.png' %>
-          <% end %>
-        </div>
+      <div class="stats col-sm-offset-2 col-sm-2">
```

Comment:
The XS is on a different grid level than the SM. You would typically want pictures to be stacked on SM and XS grid levels, then maybe two wide on MD, then four on LG